<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionSettings</class>
 <widget class="QDialog" name="ConnectionSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configurar Conex√£o</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>453</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Host:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="input_host"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Porta:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="input_port">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Database:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="input_database"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="input_username"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Senha:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="input_password">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="button_disconn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&amp;Desconectar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_conn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Co&amp;nectar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="button_clear">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&amp;Limpar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_quit">
        <property name="text">
         <string>&amp;Fechar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="manter_conectado">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Mantenha-me conectado</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>input_port</tabstop>
  <tabstop>input_database</tabstop>
  <tabstop>input_username</tabstop>
  <tabstop>input_password</tabstop>
  <tabstop>button_conn</tabstop>
  <tabstop>button_disconn</tabstop>
  <tabstop>button_clear</tabstop>
  <tabstop>button_quit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_quit</sender>
   <signal>released()</signal>
   <receiver>ConnectionSettings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
